Fitter/scripts/EFTFitter.py

```diff
@@ -147,13 +147,33 @@ def log_subprocess_output(self, pipe, level):
     #        if level=='info': logging.info(line.rstrip('\n'))
     #        if level=='err': logging.error(line.rstrip('\n'))
 
+    def __override_CMSSW_BASE(self):
+        """If CMSSW_BASE points to an /afs/ path but cwd is under /users/,
```

Comment:
This is a nice function, but really people should be moving their CMSSW builds from `/afs/.` to `/users/` and run `scram b ProjectRename` to fix the dependencies.